docs/docs/getting-started/observability.md

```diff
@@ -474,122 +474,91 @@ View the Keptn Applications Dashboard and you should see the DORA metrics and an
-    - **name** - name of this Kubernetes
-      [workload](https://kubernetes.io/docs/concepts/workloads/).
-      Use the same naming rules listed above for the application name.
-      Provide one entry for each workload
-      associated with this Keptn application.
-    - **version** -- version number for this workload.
-      Changing this number causes a new execution
-      of checks for this workload only,
-      not the entire application.
-
-The remaining fields are required only when implementing
-the release lifecycle management feature.
-If used, these fields must be populated manually:
-
-- **preDeploymentTasks** -- list each task
-    to be run as part of the pre-deployment stage.
-    Task names must match the value of the `metadata.name` field
-    for the associated [KeptnTaskDefinition](../reference/crd-reference/taskdefinition.md) resource.
-- **postDeploymentTasks** -- list each task
-    to be run as part of the post-deployment stage.
-    Task names must match the value of the `metadata.name` field
-    for the associated
-    [KeptnTaskDefinition](../reference/crd-reference/taskdefinition.md)
-    resource.
-- **preDeploymentEvaluations** -- list each evaluation to be run
-    as part of the pre-deployment stage.
-    Evaluation names must match the value of the `metadata.name` field
-    for the associated
-    [KeptnEvaluationDefinition](../reference/crd-reference/evaluationdefinition.md)
-    resource.
-- **postDeploymentEvaluations** -- list each evaluation to be run
-    as part of the post-deployment stage.
-    Evaluation names must match the value of the `metadata.name` field
-    for the associated [KeptnEvaluationDefinition](../reference/crd-reference/evaluationdefinition.md)
-    resource.
-
-## Example
+This span link needs to be added the the `KeptnAppContext`, which will ensure the connection
+between traces of different versions of the application.
 
 ```yaml
-apiVersion: lifecycle.keptn.sh/v1alpha3
-kind: KeptnApp
+apiVersion: lifecycle.keptn.sh/v1beta1
+kind: KeptnAppContext
 metadata:
-  name: podtato-head
-  namespace: podtato-kubectl
+  name: keptndemoapp
+  namespace: keptndemo
 spec:
-  version: "latest"
-  workloads:
-  - name: podtato-head-left-arm
-    version: "my_vers12.5"
-  - name: podtato-head-left-leg
-    version: "my_v24"
-  postDeploymentTasks:
-  - post-deployment-hello
-  preDeploymentEvaluations:
-  - my-prometheus-definition
+  metadata:
+    commit-id: "1234"
+    author: "myUser"
+  spanLinks:
+    - "00-c088f5c586bab8649159ccc39a9862f7-f862289833f1fba3-01"
 ```
 
-You may have noticed that the `KeptnApp` Custom Resources are created automatically by Keptn.
+> **Note**
+Please be aware that the span link is just an example and you need to retrieve the traceID and spanID
+of the `keptndemoapp-0.0.4` application trace.
+
+In the end, you need to bump the version of your `keptndemoapp` application to version `0.0.5`, redeploy
+the application as well as `KeptnAppContext` and
```

Comment:
Does KeptnAppContext also have version and revision?  I don't see it.

Reply:
no it does not, only KeptnApp

Reply:
Oh, I see, you're just saying to redeploy KeptnAppContext; I was reading it as needing o bump the version in KeptAppContext.  Sorry.